apps/src/studio/levels.js

```diff
@@ -2977,16 +2977,14 @@ levels.js_hoc2015_change_setting = {
   ],
   'progressConditions' : [
     // Collected all the items and set the right properties?  Success.
-    { required: { 'setMap': true, 'setDroidSpeed': true, 'collectedItemsAtOrAbove': 3 },
+    { required: { 'setDroid': true, 'setDroidSpeed': true, 'collectedItemsAtOrAbove': 3 },
```

Comment:
doh, great catch.  Should be fixed now, and tested in both blocks & non-blocks.  Thanks!